cap/modules/theme/templates/cap_theme/home.html

```diff
@@ -99,7 +99,7 @@ <h3 class="pull-left">Our mission is to preserve the analyses <br> across all CE
         </div>
         <div class="row">
           <div class="col-md-12">
-             <a id="login-button"  href=" {{url_for('invenio_oauthclient.login', remote_app='cern', next=url_for('cap_theme.index'))}}" class="btn btn-primary btn-lg" >{{_('Log in with your CERN account')}}</a>
+             <a id="home-login-button" href=" {{url_for('invenio_oauthclient.login', remote_app='cern', next=url_for('cap_theme.index'))}}" class="login-button btn btn-primary btn-lg" >{{_('Log in with your CERN account')}}</a>
```

Comment:
You can remove completely the id and just use the login-button class as it was before.